scripts/area_plot.py

```diff
@@ -0,0 +1,613 @@
+import argparse
+import warnings
+# from warnings import Warning
+
+from matplotlib import rcParams
+from matplotlib import cm
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+"""
+A set of python scripts to generate an area plot.
+"""
```

Comment:
Please add something like these lines
```suggestion
"""
__author__ = "Justine Debelius"
__date__ = "2020"
__version__ = "1.0"
```